defaults/main.yml

```diff
@@ -1,42 +1,33 @@
 ---
-apache_listen_ip: "*"
+
+apache_listen_ip: '*'
 apache_listen_port: 80
 apache_listen_port_ssl: 443
 
-apache_create_vhosts: true
-apache_vhosts_filename: "vhosts.conf"
-apache_vhosts_template: "vhosts.conf.j2"
-apache_default_vhost_filename: "000-default"
+apache_vhosts_version: '2.4'
```

Comment:
is this working on Debian and EL out of the box or is it neccessary to adjust it for one of the distros?

Reply:
This is works on Debian9 and Centos7 out of the box, I tested locally with [debian9](https://app.vagrantup.com/generic/boxes/debian9) and `libvirt-adsy-centos-7.4` boxes. Mainly, `apache_vhosts_version` is used in [`templates/vhosts.conf.j2`](https://github.com/adfinis-sygroup/ansible-role-apache/blob/master/templates/vhosts.conf.j2) to configure [access control](https://httpd.apache.org/docs/2.4/upgrading.html#access) directives for the specific version, `2.2` or `2.4`. However, this variable is redundant with Debian9 and Centos7 anyway, because `apache-2.4` will be installed by default.